### ISSUE TITLE
Bug "Kaleidoscope"

### ISSUE THREAD

**Issue by Romashka675** [ValveSoftware/Dota2-Gameplay#11178]
### Description

The screenshots below show how the textures of the map magically appear in the air and overlap each other, creating a kaleidoscope effect. This is not a problem with my computer, other people with good computers have this, please fix it. I can't play because of this.

### Screenshots

![Снимок экрана 2023-08-31 в 14 23 07](https://github.com/ValveSoftware/Dota2-Gameplay/assets/143745176/675bc143-88e7-4a13-b6de-ceea3eacc798)
![Снимок экрана 2023-08-31 в 14 23 00](https://github.com/ValveSoftware/Dota2-Gameplay/assets/143745176/30105a73-e622-4d2e-8e13-99acd3eedd31)
![Снимок экрана 2023-08-31 в 14 23 26](https://github.com/ValveSoftware/Dota2-Gameplay/assets/143745176/4454721b-89d1-4da3-a2d7-76b6babd552e)
![Снимок экрана 2023-08-31 в 14 23 10](https://github.com/ValveSoftware/Dota2-Gameplay/assets/143745176/36534fe8-9d58-483f-924e-3067a64cf833)

**Comment by Golf129** [comment]
Are you using MacOS? There are other reports in #11136 #11171 #11172

**Comment by Romashka675** [comment]
yes, I use mac Os, but I think it's unlikely because of this, because I've seen the same problem on windows.

> Are you using MacOS? There are other reports in #11136 #11171 #11172

**Comment by 4erepaxen** [comment]
check https://github.com/ValveSoftware/Dota2-Gameplay/issues/11361#issuecomment-1703915017
it help

**Comment by Romashka675** [comment]
> check [#11361 (comment)](https://github.com/ValveSoftware/Dota2-Gameplay/issues/11361#issuecomment-1703915017) it help



Thank you bro, you are the best
